tests/expectations/tests/objc_class.rs

```diff
@@ -9,6 +9,11 @@
 extern crate objc;
 #[allow(non_camel_case_types)]
 pub type id = *mut objc::runtime::Object;
+type Class = *mut objc::runtime::Class;
+#[inline]
+fn class(name: &str) -> Class {
+    unsafe { std::mem::transmute(objc::runtime::Class::get(name)) }
```

Comment:
Well, yes, it can be inline, I'm mostly just imitating what cocoa-rs is doing for now.
The expect is a bit different, though might be better in this case.

Normally NSClassFromString can return nil, you either check for that or rely on the silent nil for it to do nothing interesting, but in this case knowing that the call doesn't do anything might be more useful, and have the dynamic availability checks be done other ways..

I'll do it like you suggested for now, but I'm pretty sure that the generated code will have various revisions after I get some real code running